website/src/main/tut/docs/users/faq.md

```diff
@@ -10,6 +10,136 @@ If you have any questions, don't hesitate to ask on {% gitter %}.
 * TOC
 {:toc}
 
+## How does Scalafix compare with alternatives?
+
+There are many alternative tools to Scalafix, that each make different
+tradeoffs with regards to Scala 2.x fidelity, ease of writing custom
+analyses, interactivity, performance, integrations and feature support.
+The table below provides a rough comparison, below are more detailed
+explanations.
+
+|              | Scalafix       | IntelliJ Scala | Scala Refactoring | WartRemover    | ScalaStyle  |
+| -------      | --------       | -------------- | ----------------- | -----------    | ----------  |
+| Syntax model | Scalameta      | IntelliJ Scala | Scala Compiler    | Scala Compiler | Scalariform |
+| Typechecker  | Scala Compiler | IntelliJ Scala | Scala Compiler    | Scala Compiler | n/a         |
+| Linting      | Yes            | Yes            | Yes               | Yes            | Yes         |
+| Refactoring  | Yes            | Yes            | Yes               | No             | No          |
+
+### IntelliJ Scala
+
+The IntelliJ Scala Plugin is probably the most used IDE for Scala development,
+supporting a wide range of features to browse/edit/refactor Scala code.
+
+* Scalafix uses the Scala 2.x compiler to resolve symbols/types while
+  the IntelliJ Scala plugin uses it's own own typechecker.
+  This means that if a project compiles with the Scala compiler,
+  then scalafix can analyze the source code (assuming no conflicting compiler plugins).
+  The IntelliJ Scala typechecker is known to produce false red squigglies.
+  It depends on what Scala features you use whether the IntelliJ Scala can analyze
+  your source code.
+* Scalafix is primarily aimed to be used in batch-mode through a console interface
+  while IntelliJ is primarily aimed for interactive use in the IntelliJ IDE.
```

Comment:
Can Scalafix be used to integrate with an LSP editor-client linting/code-actions?

Reply:
Yes it can :) https://github.com/scalameta/language-server/pull/1

Reply:
Oh my!!! I was just thinking exactly about this project last days! Great 👍 👍 👍  I'm really glad you are already working on it. I'm going to follow development closely.